test/unit/auth/user-import-builder.spec.ts

```diff
@@ -293,23 +328,59 @@ describe('UserImportBuilder', () => {
           });
         });
 
-        it('should not throw with valid options and should generate expected request', () => {
-          const validOptions = {
-            hash: {
-              algorithm,
+        if (algorithm !== 'PBKDF_SHA1' && algorithm !== 'PBKDF2_SHA256' && algorithm !== 'MD5') {
```

Comment:
Why are you excluding these algorithms specifically?

Reply:
We don't support specifying hashing order for 'PBKDF_SHA1', 'MD5', and 'PBKDF2_SHA256' algorithms. Inside the if statement we're only testing special cases for password hash order supported algorithms.

Reply:
Hashing order is not supported for other algorithms too (eg. bcrypt, scrypt, etc)

Reply:
This condition is based on the forloop for md5ShaPbkdfAlgorithms where 
const md5ShaPbkdfAlgorithms = ['MD5', 'SHA1', 'SHA256', 'SHA512', 'PBKDF_SHA1', 'PBKDF2_SHA256'];
In the origin code structure, algorithms like bcrypt and scrypt have their own defined sections for testing because they differ a lot from others. Maybe I'll try add one extra test each for these individual algorithms.

Reply:
for your previous comment "test when hash input order is specified with algorithm that doesn't support it."

Reply:
Ah ok. The GitHub review UI was not helping. Can you just add a comment that hash input order is not supported for these algorithms?

Reply:
sure.